workers/print-puppeteer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Initially based upon:
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-FROM node:12-buster-slim
+FROM node:14-buster-slim
```

Comment:
maybe more specific with 14.15 (or direct pin to 14.15.4 and update with renovate)?